demos/action_recognition_demo/python/action_recognition_demo/steps.py

```diff
@@ -101,6 +101,9 @@ def __init__(self, encoder):
         self.encoder = encoder
         self.async_model = AsyncWrapper(self.encoder, self.encoder.num_requests)
 
+    def __del__(self):
+        self.encoder.infer_queue[0].cancel()
```

Comment:
Should we call `cancel()` for all infer requests in infer queue or is it enough only for the first infer request? @Wovchena @VoronovaIntern

Reply:
For all